src/mame/mame.lst

```diff
@@ -33107,6 +33107,7 @@ stratvox                        // [1980 Sun Electronics] Taito
 stratvoxa                       // [1980 Sun Electronics] Taito
 stratvoxb                       // bootleg
 ttmahjng                        // Taito
+jongpute                        // Alpha Denshi Co.
```

Comment:
Should this be in the dict order? I replaced this to the right place in the second commit.

Reply:
It's generally kept in alphabetical order within a file just to make things easy to find, but that's a minor thing.